packages/dashmate/src/docker/DockerCompose.js

```diff
@@ -260,6 +260,10 @@ class DockerCompose {
         commandOptions,
       }));
     } catch (e) {
+      if (e.err && e.err.startsWith('no such service:')) {
```

Comment:
Why do hide this error?

Reply:
Behaviour of the command `docker-compose ps --services` a little bit changed. Now it throws error, if it cannot find any services. And the services are only listed there after you execute up command (`docker compose up`) at least once.

Previously, this command was receiving empty array instead of error and was working fine

Reply:
I see. Thank you.